client_app/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
```

Comment:
We may need to add "npm install" command for first-time users

Reply:
Thank you @AbhilashKotha. I updated the README.md file with the instruction to run "npm install"

Reply:
I think we can Merge this now. Once we have this folder structure merged, we can commit our files to the wireframes directory.

Reply:
For the sake of following the process, can you approve the PR?